test/sql/aggregate/qualify/test_qualify_view.test

```diff
@@ -0,0 +1,53 @@
+# name: test/sql/aggregate/qualify/test_qualify_view.test
+# description: Test QUALIFY clause in a view over different runs
+# group: [qualify]
+
+# load the DB from disk
+load __TEST_DIR__/view_storage.db
+
+statement ok
+PRAGMA disable_checkpoint_on_shutdown
+
+statement ok
+PRAGMA wal_autocheckpoint='1TB';
+
+# create a schema and view
+statement ok
+CREATE SCHEMA test;
+
+statement ok
+CREATE TABLE test.t (a INTEGER, b INTEGER);
+
+statement ok
+CREATE VIEW test.v AS SELECT * FROM test.t;
```

Comment:
Can you modify the test so the view contains `QUALIFY`?

Reply:
Okay, I will modify so the view contains QUALIFY.